blog/models.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=100)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_created=True)
```

Comment:
Replace `auto_created` with `auto_now_add`, it is a common way to store the object creation timestamp.